build/rakelib/cluster.rake

```diff
@@ -82,14 +82,19 @@ namespace :cluster do
   namespace :sifnode do
     namespace :deploy do
       desc "Deploy a single standalone sifnode on to your cluster"
-      task :standalone, [:chainnet, :provider, :namespace, :image, :image_tag] do |t, args|
+      task :standalone, [:chainnet, :provider, :namespace, :image, :image_tag, :ethereum_websocket_address, :ethereum_contract_address, :ethereum_private_key] do |t, args|
```

Comment:
The `standalone` task is for single node iso testnet deployments. Petty sure this should be either a new tasks or update the `peers` tasks @intl-man whats your take?

Reply:
@ilchebedelovski this should be a completely new task, something like `ebrelayer:deploy`. We should try and keep the deployment stories for `sifnoded/sifnodecli` and `ebrelayer` seperate, as not all validators will be able to run `ebrelayer`.

Reply:
@ilchebedelovski this should be a completely new task, something like `ebrelayer:deploy`. We should try and keep the deployment stories for `sifnoded/sifnodecli` and `ebrelayer` seperate, as not all validators will be able to run `ebrelayer`.

Reply:
@ilchebedelovski this should be a completely new task, something like `ebrelayer:deploy`. We should try and keep the deployment stories for `sifnoded/sifnodecli` and `ebrelayer` seperate, as not all validators will be able to run `ebrelayer`.

Reply:
@ilchebedelovski this should be a completely new task, something like `ebrelayer:deploy`. We should try and keep the deployment stories for `sifnoded/sifnodecli` and `ebrelayer` seperate, as not all validators will be able to run `ebrelayer`.

Reply:
@intl-man @utx0 I made the `ebrelayer` deployment be configurable and I have added a separate subtask inside the `sifnode:deploy` task because we cannot move `ebrelayer` in a different namespace since we are sharing storage with `sifnodecli`.

Reply:
@ilchebedelovski Not sure that will work, because not all validators will run `ebrelayer`, so they won't have those ethereum variables to set...